Calibration/PPSAlCaRecoProducer/test/test_express_AlCaRecoProducer.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+function die { echo $1: status $2; exit $2; }
+
+customise_commands="process.GlobalTag.toGet = cms.VPSet()\
+\nprocess.GlobalTag.toGet.append(cms.PSet(record = cms.string(\"AlCaRecoTriggerBitsRcd\"),tag =  cms.string(\"AlCaRecoHLTpaths_PPS2022_express_v1\"), connect = cms.string(\"frontier://FrontierProd/CMS_CONDITIONS\")))\
```

Comment:
Are these tags still not available in the GT?

Reply:
They are, but they are added to the online GT, thus valid from 2022, i.e. it wont work on a replay with 2018 data. Everything with PPS will be just so much clearer when they collect their first data this year...